ballerina/natives.bal

```diff
@@ -123,6 +135,25 @@ public isolated function printWarn(string msg, error? 'error = (), *KeyValues ke
     }
 }
 
+# Set the log output to a file. Note that all the subsequent logs of the entire application will be written to this file.
+# ```ballerina
+# log:setOutputFile("./resources/myfile.log");
+# log:setOutputFile("./resources/myfile.log", log:OVERWRITE);
+# ```
+#
+# + path - The path of the file
+# + option - To indicate whether to overwrite or append the log output
+public isolated function setOutputFile(string path, FileWriteOption option = APPEND) {
+    lock {
+        outputFilePath = path;
+    }
```

Comment:
shall we validate the file path here?. should be a `.log` extension right?. and also if there are any errors, we should not assign the path value.

Reply:
Updated in 3038cc1